specs/0006-label-taxonomy.md

```diff
@@ -0,0 +1,137 @@
+# A Taxonomy for Issue and PR Labels
+
+This is a proposal for formalizing the labels we use in our
+repositories in order to make them more meaningful and more useful.
+
+## Problem Description
+
+We use issue and pull request labels to indicate certain metadata
+about these items. Some labels are used to position an issue along a
+scale (is this high or low priority? Is this a big or small issue?).
+Other labels are used to group together issues that share some common
+aspect ("These are website issues" or "These issues need better
+descriptions").
+
+Our collection of labels has grown organically over time, and as we
+have created new labels they have not always been propagated to all
+of our repositories. This leads to two problems:
+
+- We can't always use labels effectively in our sprint planning
+  because sometimes the labels aren't available when we want them.
+
+- It can be difficult to differentiate between different types of
+  labels.
+
+## Policy
+
+I would like to propose a common label taxonomy to adopt across all of
+our repositories. The labels will be organized around several axes,
+indicated by prefixes on the label name. For example, labels that
+indicate priority will live in the `priority` access, and will be
+named like `priority/critical` or `priority/medium`. We should start
+with the following axes that cover most of our existing labels:
+
+- **Type**
+
+  The `type` axis describes in general terms the sort of work required
+  to resolve the issue.
+
+  - `type/bug`
+  - `type/enhancement`
+  - `type/documentation`
+  - `type/research`
+  - `type/discussion`
+
+- **Status**
+
+  The `status` axis describes the current condition of the issue or
+  pull request. This is a multi-valued axis (e.g., an issue may be
```

Comment:
can other types not be multi-valued? is that going to be enforced somehow (picking one would remove other label?)

Reply:
Nope, no enforcement. Just an indication of intended use.